### ISSUE TITLE
Missing backend UI to manage shipping rules (aka checkers?)

### ISSUE THREAD

**Issue by jimbocoder** [Sylius/Sylius#2766]
There's a UI for promotions to manage the rules, but there's currently no corresponding feature for shipping rules.  It looks like someone began this feature, but it is not complete, or broken in a way that causes the rule form to not render at all.

I tried hard to finish or fix this, but I'm still becoming familiar with Symfony.  Onnit (my employer) would be extremely interested in this feature, or any advice so I can give it another try myself.

**Comment by jimbocoder** [comment]
Just leaving a note here in case someone wants to work on this.  I've got it mostly working and will eventually make a PR of it, but if I haven't done so yet, just hit me up and I will share my current progress, which is hopefully heading in the right direction.

**Comment by pjedrzejewski** [comment]
@jimbocoder Hey Jim! Let me know if you have moved forward or plan to do so in coming days/weeks, otherwise I will try to organize someone to help. :) Thanks!

**Comment by jimbocoder** [comment]
@pjedrzejewski wish I had got around to this. I've recently changed employers, so Sylius isn't part of my daily life anymore.  Hopefully someday soon that changes.  I am huge fan of what you guys are doing here! :)

I've pushed the progress I made to a new branch.  It's rough, and lacks tests, but it does "work", so I hope it's a useful starting point.  The branch is here: https://github.com/jimbocoder/Sylius/tree/shipping-rules-ui
